core/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/status/TransportNodesSnapshotsStatus.java

```diff
@@ -192,13 +192,13 @@ public NodeRequest() {
         @Override
         public void readFrom(StreamInput in) throws IOException {
             super.readFrom(in);
-            snapshotIds = in.readList(SnapshotId::readSnapshotId);
+            snapshotIds = in.readList(streamInput -> new SnapshotId(streamInput));
```

Comment:
Simpler: `snapshotIds = in.readList(SnapshotId::new);`
No need to create lambda :-)